env/env_posix.cc

```diff
@@ -1222,7 +525,9 @@ Env* Env::Default() {
   CompressionContextCache::InitSingleton();
   INIT_SYNC_POINT_SINGLETONS();
   static PosixEnv default_env;
-  return &default_env;
+  static CompositeEnvWrapper composite_env(&default_env,
+                                           FileSystem::Default().get());
```

Comment:
Is it by design that `FileSystem::Default()` always return the PosixFileSystem instance?

Reply:
Yes, along the lines of Env::Default().